README.md

```diff
@@ -33,8 +34,8 @@ EOS.IO currently supports the following operating systems:
 1. [Getting Started](#gettingstarted)
 2. [Setting up a build/development environment](#setup)
 	1. [Automated build script](#autobuild)
-      1. [Clean install Linux (Ubuntu & Fedora) for a local testnet](#autoubuntulocal)
-      2. [Clean install Linux (Ubuntu & Fedora) for the public testnet](#autoubuntupublic)
+      1. [Clean install Linux (Amazon, Fedora & Ubuntu) for a local testnet](#autoubuntulocal)
+      2. [Clean install Linux (Amazon, Fedora & Ubuntu) for the public testnet](#autoubuntupublic)
```

Comment:
A programmer who doesn't use Oxford comma. I didn't know that existed. I would vote strongly for `Amazon, Fedora, & Ubuntu`. Here and through out your changes.

Reply:
This seems to be a hotly debated topic:
https://www.google.com/search?client=safari&rls=en&q=comma+and+ampersand&ie=UTF-8&oe=UTF-8

Reply:
As requested, commas have been added, & Submodules updated.